core/issuers.js

```diff
@@ -35,6 +65,21 @@ Description must be truthy: ${description}`;
       getBalance() {
         return mintController.getAmount(payment);
       },
+      getUse() {
+        return makeUseObj(issuer, payment);
+      },
+      getDelegatedUse() {
+        // quantity is the same, but amounts are different for
+        // different issuers
+        const childAmount = getChildAmount(amount);
+        const childPurse = childMint.mint(childAmount);
+        const childPayment = childPurse.withdrawAll();
+        return makeUseObj(childIssuer, childPayment);
+      },
+      revokeChildren() {
+        const childAmount = getChildAmount(amount);
+        childMint.destroy(childAmount);
+      },
     });
```

Comment:
Just realized that the `payment` object definition is also free of side effects, so the "commit point" comment should be moved below it, here.

Reply:
got it!